src/core/TermBuilder.ml

```diff
@@ -68,6 +68,22 @@ let el_in m : _ m =
   let+ tm = m in
   S.ElIn tm
 
+let tp_wrap_prf mphi : _ m = 
+  let+ phi = mphi in
+  S.TpLockedPrf phi
+
+let wrap_prf_in mprf : _ m = 
+  let+ prf = mprf in 
+  S.LockedPrfIn prf 
```

Comment:
I wonder why `tp_wrop_prf` and `wrap_prf_in` are never used...?

Reply:
These are not used because they were not needed anywhere yet. but I add them for completeness.

I will fix the names, thanks!